### PR TITLE
Add bufbuild/protoschema-plugins (bigquery, pubsub, jsonschema) v0.1.0

### PR DESCRIPTION
**Pull request by mfridman**
Fix https://github.com/bufbuild/plugins/issues/1113

This PR adds 3 plugins:

- [x] buf.build/bufbuild/protoschema-pubsub
- [x] buf.build/bufbuild/protoschema-bigquery
- [x] buf.build/bufbuild/protoschema-jsonschema

### CODE REVIEW

File: plugins/bufbuild/protoschema-pubsub/v0.1.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# syntax=docker/dockerfile:1.6
+FROM --platform=$BUILDPLATFORM golang:1.22.1-bookworm AS build
+
+ARG TARGETOS TARGETARCH
```
Comment (mfridman) [inline plugins/bufbuild/protoschema-pubsub/v0.1.0/Dockerfile:4]: cc @pkwarren is this what you had in mind?

File: plugins/bufbuild/protoschema-pubsub/v0.1.0/buf.plugin.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+version: v1
+name: buf.build/bufbuild/protoschema-pubsub
+plugin_version: v0.1.0
+source_url: https://github.com/bufbuild/protoschema-plugins
+description: Generates a PubSub schema in the form of a single self-contained messaged normalized to proto2.
+registry:
+  archive: {}
```
Comment (mfridman) [inline plugins/bufbuild/protoschema-pubsub/v0.1.0/buf.plugin.yaml:7]: cc @cyinma can you check the registry config bit?

Comment (cyinma) [reply plugins/bufbuild/protoschema-pubsub/v0.1.0/buf.plugin.yaml:7]: yes, this matched how it looks like in [bufbuild/buf](https://github.com/bufbuild/buf/blob/main/private/bufpkg/bufplugin/bufpluginconfig/testdata/success/archive/buf.plugin.yaml#L8-L9) too

File: plugins/bufbuild/protoschema-pubsub/v0.1.0/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+ENV CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH 
+
+RUN --mount=type=cache,target=/go/pkg/mod \
+    go install -ldflags="-s -w" -trimpath github.com/bufbuild/protoschema-plugins/cmd/protoc-gen-pubsub@v0.1.0
```
Comment (pkwarren) [inline plugins/bufbuild/protoschema-pubsub/v0.1.0/Dockerfile:8]: I think you need to add `mv /go/bin/${GOOS}_${GOARCH}/protoc-gen-pubsub /go/bin/protoc-gen-pubsub || :` after this completes (if it exists). Try running `docker build --platform linux/amd64 .` to see the error without it.

**Review by pkwarren — CHANGES_REQUESTED** [review]

**Review by pkwarren — APPROVED** [review]

**Pull request merged**